src/libbacktrace/Makefile.am

```diff
@@ -6,12 +6,12 @@
 # met:
 
 #     (1) Redistributions of source code must retain the above copyright
-#     notice, this list of conditions and the following disclaimer. 
```

Comment:
Thank you for your comments! This change can be abandoned and be split for easier reviewing. Sorry about making a big change. Of course if trailing whitespaces are not a problem at all, then change does not have to be made in first place.